eventing/event-sources/serverless-target-source-odc.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
```

Comment:
BTW, the [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata) have the include under the id, and I have no idea whether the order of lines makes any difference here.

Reply:
I don't think it matters really - I'll leave it as is for now since technically I didn't create the file, just renamed it 🙂